contributing-guides/style-guide.md

```diff
@@ -468,19 +468,17 @@ Keep the following guidelines in mind when choosing placeholders:
 
 - If a command can optionally take 1 or more arguments of the same kind, use an ellipsis: `{{placeholder1 placeholder2 ...}}`.
   For instance, if multiple paths are expected, use `{{path/to/directory1 path/to/directory2 ...}}`.
-- If a command can optionally take 1 or more arguments of different kinds, use an ellipsis: `{{placeholder1|placeholder2|...}}`.
-  If there are more than 5 possible values, you can use `|...` after the last item.
-- It's impossible to restrict the minimum or (and) maximum placeholder count via `ellipsis`.
-
-It's up to the program to decide how to handle duplicating values, provided syntax
-tells no info about whether items are mutually exclusive or not.
+- If only one of the multiple options is possible, write it as: `{{placeholder1|placeholder2|placeholder3}}`. If there are more than 4 possible values, you can use `|...` after the last item.
 
 #### Optional placeholders
 
 When documenting optional placeholders like paths or file extensions, it is suggested to specify them in the page or example descriptions instead of the placeholder itself. For example:
 
 - Use `{{path/to/source.ext}}` instead of `{{path/to/source.tar[.gz|.bz2|.xz]}}`.
 
+> [!NOTE]\
+> Angle brackets `[]` must always be a substring of the placeholder to not get mixed up with an option placeholder.
```

Comment:
```suggestion
> Angle brackets `[]` must always be within the placeholder in order not to get mixed up with an option placeholder.
```

Reply:
This is not good enough. {{[string]}} is still within the placeholder

Reply:
What exactly would you like to address with this PR? Could you give an example? I am not sure I understand what the problem is

Reply:
Since option placeholders will soon be a thing, {{[]}} will be reserved for those. To make sure pages don't use this pattern when long and short options are not involved.

Reply:
I think we need to find a different wording then. In my opinion, "substring" implies that the literal sequence`[]` should appear (and not, e.g. `[something]`).

Reply:
I get what you mean. Please suggest something. I'm in bed and only responding to emails.

Reply:
I wasn't able to come up with a good rewording so I removed the note. Lets reconsider this if this becomes a problem.

Reply:
Me too unfortunately